docs/QuestionsIssuesPrs.md

```diff
@@ -38,7 +38,7 @@ If you don't get an answer within 1-2 days, feel free to post another comment in
 
 ### PR workflow
 
-1. **Select an issue to fix** 
+1. #### Select an issue to fix
```

Comment:
It is documented in the recent [GitHub Flavoured Markdown Spec](https://github.github.com/gfm/#example-268).